### ISSUE TITLE
DL4J UI: Shade freemarker library

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#8750]
Used only in DL4J UI.
To avoid version incompatibilities like this: https://github.com/eclipse/deeplearning4j/issues/8725

**Comment by AlexDBlack** [comment]
Attempted to simply shade freemarker library in this PR: https://github.com/KonduitAI/deeplearning4j/pull/318

Problem is an obscure resource loading issue on linux-x86_64 CI.
Not reproducible locally (windows, linux).
Not reproducible using the exact same docker image as CI uses.

```
[2020-03-20T11:12:18.897Z] java.lang.ExceptionInInitializerError
[2020-03-20T11:12:18.897Z] 	at org.deeplearning4j.ui.TestStandAlone.testStandAlone(TestStandAlone.java:58)
[2020-03-20T11:12:18.897Z] Caused by: java.lang.RuntimeException: Failed to load and parse /freemarker/version.properties
[2020-03-20T11:12:18.897Z] 	at org.deeplearning4j.ui.TestStandAlone.testStandAlone(TestStandAlone.java:58)
[2020-03-20T11:12:18.897Z] Caused by: java.io.IOException: Class-loader resource not found (shown quoted): "/org/nd4j/shade/freemarker/version.properties". The base class was org.nd4j.shade.freemarker.template.Configuration.
[2020-03-20T11:12:18.898Z] 	at org.deeplearning4j.ui.TestStandAlone.testStandAlone(TestStandAlone.java:58)
[2020-03-20T11:12:18.898Z] 
[2020-03-20T11:12:19.473Z] Running org.deeplearning4j.ui.TestComponentSerialization
```

The apparently missing file `/org/nd4j/shade/freemarker/version.properties` definitely exists in the JAR file.

Blocked on this issue until this can be resolved (alreday spent a fair bit of time, no luck so far)

**Comment by saudet** [comment]
@sshepel Any ideas?

**Comment by sshepel** [comment]
Hm, no... will try to debug.

**Comment by agibsonccc** [comment]
Closing — this issue is over 5 years old with no recent activity. The codebase has been significantly rewritten. If this is still relevant, please open a new issue.
